src/api/app/views/webui/project/subprojects.html.haml

```diff
@@ -10,12 +10,12 @@
     %thead
       %tr
         %th Parent project
-        %th Description
+        %th Title
     %tbody
       - @parentprojects.each do |project|
         %tr
-          %td= link_to project.name, :action => 'show', :project => project.name
-          %td= project.title
+          %td{:width => "50%"}= link_to project.name, :action => 'show', :project => project.name
```

Comment:
Maybe project name could be also word wrapped and there could be defined max widths for columns, so that it wouldn't exceed content container? For example https://build.opensuse.org/project/subprojects/home:kulmesa